integrations/cdi/hibernate-cdi/src/main/java/io/helidon/integrations/cdi/hibernate/HibernateJtaPlatformProvider.java

```diff
@@ -30,14 +30,13 @@
  *
  * @see JtaPlatform
  *
- * @see CDISEJtaPlatform
+ * @see HibernateJtaPlatform
  */
-public final class CDISEJtaPlatformProvider implements JtaPlatformProvider {
-
+public final class HibernateJtaPlatformProvider implements JtaPlatformProvider {
   /**
-   * Creates a new {@link CDISEJtaPlatformProvider}.
+   * Creates a new {@link HibernateJtaPlatformProvider}.
```

Comment:
`JtaPlatformProvider` is inherently a Hibernate construct so renaming this to `HibernateJtaPlatformProvider` is redundant.

Reply:
Reverted